flowey/flowey_core/src/pipeline.rs

```diff
@@ -1140,6 +1140,16 @@ impl PipelineJob<'_> {
             job_idx: self.job_idx,
         }
     }
+
+    /// Return the job's platform.
```

Comment:
I don't see these being consumed by this PR. Is that intentional?

Reply:
In isolation, yes, it seems weird that these APIs are being added as part of this PR, and should _probably_ be organized under a separate PR.

...that said, given that this PR was written in support of an internal PR, it seems fine to have them tag-along with the rest of these changes.